solar/solar/core/resource/resource.py

```diff
@@ -270,6 +270,7 @@ def _connect_inputs(self, receiver, mapping):
             mapping = dict((x, x) for x in mapping)
         self.db_obj.connect(receiver.db_obj, mapping=mapping)
         self.db_obj.save_lazy()
+        receiver.db_obj.save()
```

Comment:
Could you please change it to `save_lazy()` ? Session issues are now fixed, so it should be fine.